src/numerics.jl

```diff
@@ -33,6 +35,22 @@ function compute_equilibrium!(discretization; max_years=100, rel_error=2e-5, ver
     for year in 1:max_years
         average_temperature = 0.0
         for time_step in 1:num_steps_year
+
+            # monthly update starting with first week of April
+            if mod(time_step,4) == 2 && time_step in (2:48)
+                if update_heat_capacity == true && update_solar_forcing == true     
+                 update_model(model, mesh,time_step, true,true,false,true,true)
+                 lu_decomposition = compute_lu_matrices(mesh, model, num_steps_year)
+                elseif update_heat_capacity == true && update_solar_forcing == false 
+                    update_model(model, mesh, time_step, true,false,false,true,false)
+                    lu_decomposition = compute_lu_matrices(mesh, model, num_steps_year)   
+                elseif update_heat_capacity == false && update_solar_forcing == true
+                    update_model(model, mesh, time_step, true,true,false,false,true)  
+                elseif update_heat_capacity == false && update_solar_forcing == false 
+                    update_model(model, mesh, time_step, false,false,false,false,false)
+                end
+            end    
```

Comment:
Please add this code in a function and call it from `function compute_equilibrium!` and `function compute_evolution!`